package.json

```diff
@@ -4,13 +4,12 @@
   "description": "Making company travel and accomodation easy and convinient",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "npm run build && node dist/index.js",
+    "test": "mocha --require @babel/register src/tests/*.js",
+    "start": "nodemon --exec node -r dotenv/config ./src/index.js",
     "migrate": "npx sequelize-cli db:migrate",
     "migrate:undo": "npx sequelize-cli db:migrate:undo:all",
     "watch": "nodemon --exec babel-node ./src/index.js",
-    "build": "babel src -d dist",
-    "lint": "eslint --fix ./"
+    "build": "node -r dotenv/config ./src/index.js"
```

Comment:
@gildniy  you edited the scripts that are used run the API on Heroku. like build and start they should not have nodemon because they are used during production

Reply:
> Great job so far but you edited the scripts that are used run the API on Heroku. like build and start they should not have nodemon because they are used during production

Thanks for pointing out. Now fixed!

Reply:
Awesome